js/ui/file_manager/ui.file_manager.command_manager.js

```diff
@@ -15,52 +16,62 @@ export class FileManagerCommandManager {
                 name: "create",
                 text: "New folder",
```

Comment:
You should use the localization constants of DevExtreme for a text of command. For example: `editRow: messageLocalization.format("dxDataGrid-editingEditRow")`